Eatery/Models/EaterySwipeData.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  EaterySwipeData.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 5/5/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+
+class EaterySwipeData {
+    
+    private var swipeDataByHours = [Int : [SwipeDataPoint]]()
+  
+    init(swipeDataPoints: [SwipeDataPoint]) {
+        mapSwipeDataPointToHours(swipeDataPoints: swipeDataPoints)
+    }
+    
+    private func mapSwipeDataPointToHours(swipeDataPoints: [SwipeDataPoint]) {
+        for swipeDataPoint in swipeDataPoints {
+            if swipeDataForHour(hour: swipeDataPoint.militaryHour) == nil {
+                swipeDataByHours[swipeDataPoint.militaryHour] = [swipeDataPoint]
+            } else {
+                swipeDataByHours[swipeDataPoint.militaryHour]!.append(swipeDataPoint)
+            }
+        }
+    }
+    
+    func swipeDataForHour(hour: Int) -> [SwipeDataPoint]? {
+        return swipeDataByHours[hour]
+    }
+    
+    func waitTimeFor(hour: Int, minute: Int) -> (waitTimeLow: Int, waitTimeHigh: Int)? {
```

Comment:
We can rename this:
`waitTime(forHour hour: Int, minute: Int) -> (low: Int, high: Int)`

There's no need to namespace the return tuple twice especially when the method name already has `waitTime` in its name. Moreover, it is swift style to include the preposition inside the method parameters.

Reply:
might also be worth making `(waitTimeLow: Int, waitTimeHigh: Int)` into a `typealias` of some named type